libsqlite3-sys/build.rs

```diff
@@ -108,6 +108,10 @@ mod build_bundled {
         if cfg!(feature = "session") {
             cfg.flag("-DSQLITE_ENABLE_SESSION");
         }
+        if !cfg!(feature = "bundled-complete") {
```

Comment:
This name is confusing as it actually makes the bundled build less complete -- it disables things.

Reply:
Note the `!` in front, things are disabled if this feature is omitted, not if it is enabled (which it is by default).